library-no-op/build.gradle

```diff
@@ -21,6 +21,7 @@ artifacts {
 dependencies {
     implementation "com.squareup.okhttp3:okhttp:$okhttp3Version"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
+    implementation "androidx.fragment:fragment:$fragmentVersion"
```

Comment:
Maybe we could think about some other way of implementing feature management, which would allow to not include this dependency?